src/containers/edit-profile/professional-info-tab/ProfessionalInfoTab.tsx

```diff
@@ -150,20 +212,7 @@ const ProfessionalInfoTab = () => {
           openProfessionalCategoryModal={openProfessionalCategoryModal}
         />
       </Box>
-      <Box component='section'>
-        <TitleWithDescription
-          description={t(
-            'editProfilePage.profile.professionalTab.aboutTheTutorDescription'
-          )}
-          style={styles.titleWithDescription}
-          title={t(
-            'editProfilePage.profile.professionalTab.aboutTheTutorTitle'
-          )}
-        />
-        <Box sx={styles.accordionContainer}>
-          <AboutTutorAccordion />
-        </Box>
-      </Box>
+      {userRole === UserRoleEnum.Tutor && TutorInfo}
```

Comment:
Maybe would be better to move it before return statement. What do you think about?

Reply:
Maybe you right, thank you! Solved